molecule/vault/group_vars/vault/public.yml

```diff
@@ -0,0 +1,111 @@
+---
+
+#
+# self signed certificate generated as follows for test purposes
+#
+# cat /tmp/l
+#
+#        [ req ]
+#        default_bits           = 2048
+#        default_keyfile        = keyfile.pem
+#        distinguished_name     = req_distinguished_name
+#        attributes             = req_attributes
+#        prompt                 = no
+#        output_password        = mypass
+#
+#        [ req_distinguished_name ]
+#        C                      = GB
+#        ST                     = Test State or Province
+#        L                      = Test Locality
+#        O                      = Organization Name
+#        OU                     = Organizational Unit Name
+#        CN                     = 127.0.0.1
+#        emailAddress           = test@email.address
+#
+#        [ req_attributes ]
+#        challengePassword              = A challenge password
+#
+# openssl req -config /tmp/l '-nodes' '-new' '-x509' '-keyout' 'server.key' '-out' 'server.crt'
```

Comment:
Just for your interest, I used https://github.com/cloudflare/cfssl to generate all the certs. I've felt that the config files it uses are a bit more friendly than OpenSSL's.

Reply:
thanks for the tip!